SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -1305,11 +1309,11 @@ void ActorSystem::ApplyComponentUpdate(const Worker_ComponentId ComponentId, Sch
 
 	ComponentReader Reader(NetDriver, RepStateHelper.GetRefMap(), NetDriver->Connection->GetEventTracer());
 	bool bOutReferencesChanged = false;
-	FObjectRepNotifies& ObjectRepNotifiesOut = GetObjectRepNotifies(TargetObject);
 
 	const bool bSuccessfullyPreNetReceived = InvokePreNetReceive(TargetObject);
 	if (bSuccessfullyPreNetReceived)
 	{
+		FObjectRepNotifies& ObjectRepNotifiesOut = GetObjectRepNotifies(TargetObject);
```

Comment:
why was this moved?

Reply:
Essentially a drive by - I wanted to restrict the scope of what we're calling repnotifies on as much as reasonable